Makefile

```diff
@@ -36,4 +36,7 @@ run:
 	./bin/grafana-server
 
 protoc:
-	protoc -I pkg/tsdb/models pkg/tsdb/models/*.proto --go_out=plugins=grpc:pkg/tsdb/models/.
\ No newline at end of file
+	protoc -I pkg/tsdb/models pkg/tsdb/models/*.proto --go_out=plugins=grpc:pkg/tsdb/models/.
```

Comment:
this should not longer be needed, no proto files in main repo, they are in seperate repo

Reply:
It's still present in master, the only reason it's marked as a diff here is that in master Makefile is missing a newline at the end of the last line.